### ISSUE TITLE
[FEATURE REQUEST] Save Bounce Messages

### ISSUE THREAD

**Issue by SnejPro** [knadh/listmonk#2066]
**Is your feature request related to a problem? Please describe.**
- I'd like to check why emails bounces to verify that my email systems work correctly

**Describe the solution you'd like**
- listmonk should preserve the bounce messages after processing them

**Comment by MaximilianKohler** [comment]
https://demo.listmonk.app/admin/subscribers/bounces ?

**Comment by knadh** [comment]
> https://demo.listmonk.app/admin/subscribers/bounces ?

+1. The entire message body isn't stored, but the bounce metadata/headers are stored as a JSON map for debugging.

**Comment by SnejPro** [comment]
> +1. The entire message body isn't stored, but the bounce metadata/headers are stored as a JSON map for debugging.

Correct. Most of the bounce-messages are generated by my mail-server, so the headers are usually looking the same. But the error messages replied from the recipient mail server are in the body.

**Comment by Diverti** [comment]
+1 would be great to check some bounced emails body.

**Comment by filipesmedeiros** [comment]
I would appreciate this as well, for debugging purposes
